Formula/vim@7.4.rb

```diff
@@ -77,9 +77,18 @@ def install
       opts << "--without-x"
     end
 
-    if build.with? "luajit"
```

Comment:
I don't understand why you'd want to keep a problematic formula problematic. That seems counterintuitive to Core not becoming a garbage fire.